include/pybind11/numpy.h

```diff
@@ -328,6 +331,17 @@ template <typename T> using is_pod_struct = all_of<
     satisfies_none_of<T, std::is_reference, std::is_array, is_std_array, std::is_arithmetic, is_complex, std::is_enum>
 >;
 
+// Replacement for std::is_pod (deprecated in C++20)
+#if !defined(__GNUG__) || defined(_LIBCPP_VERSION) || defined(_GLIBCXX_USE_CXX11_ABI)
+template <typename T> using is_pod = all_of<
+    std::is_standard_layout<T>,
+    std::is_trivially_constructible<T>
+>;
+#else
+template <typename T>
+using is_pod = std::is_pod<T>;
```

Comment:
It's simpler to just never use `std::is_pod`.

Reply:
GCC 4.8/4.9 are missing alternative.

Reply:
Not according to compiler explorer. Even gcc 4.5 had both `is_trivial` and `is_standard_layout`.

https://godbolt.org/z/5jKx9Y

Reply:
Ah, I was using `is_trivially_constructable`